test/run-openshift-remote-cluster

```diff
@@ -21,13 +21,13 @@ test_php_imagestream
 
 trap ct_os_cleanup EXIT SIGINT
 
-ct_os_check_compulsory_vars || exit $OC_ERR
```

Comment:
Why did you remove this?

Reply:
Upss, sorry it looks like typo. I will fix it. Thanks.

Reply:
Fixed